fed/barriers.py

```diff
@@ -110,6 +110,8 @@ async def _run_grpc_server(
     else:
         server.add_insecure_port(f'[::]:{port}')
 
+    msg = f"Successfully adding port {port}."
+    server_future.set_result((True, msg))
```

Comment:
`server_future.set_result((True, msg))` should come after `await server.start()`.